cypress/support/commands.ts

```diff
@@ -125,7 +126,13 @@ Cypress.Commands.add('drag', (selector, pos, options) => {
 				pageY: newPosition.y,
 				force: true,
 			});
-			element.trigger('mouseup', {force: true});
+			if (isDraggingANode) {
```

Comment:
Can you make it a bit more reusable? Similar to `realMouse`
You could send an option to force a click on the body and it should not be a `node` specific

Reply:
Sure, I wasn't sure should we add another option to command or just extend the logic, but will go for the option then 👍